rosdep/python.yaml

```diff
@@ -8040,6 +8040,15 @@ python3-virtualserialports-pip:
   ubuntu:
     pip:
       packages: [PyVirtualSerialPorts]
+python3-voluptuous:
+  debian: [python-voluptuous]
+  fedora: [python-voluptuous]
```

Comment:
I feel like in both of these cases, this should be `python3-voluptuous`.  Certainly on Debian that is the correct package name: https://packages.debian.org/buster/python3-voluptuous .  On Fedora, the situation is a little more murky, but if I do `dnf -y install python3-voluptuous` there, I get a valid package (and it looks like `python-voluptuos` is just a link to that).  So I'd suggest changing both of those.

Reply:
OK I think I understand your recommendation clearly, moved debian and fedora to `python3-voluptuous`.